### PR TITLE
W-17067958-rtf-path-routing-sample-kt

### PR DESCRIPTION
**Pull request by kevintroller**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: modules/ROOT/pages/custom-ingress-configuration.adoc [hunk 1]
```diff
@@ -114,8 +114,6 @@ If you add the TLS section with a `secretName` attribute, you must ensure that y
 
 <7> These placeholder values are required for Kubernetes validation, but the actual values are not used by Runtime Fabric. 
 
-For a path-based routing ingress template, consult your ingress controller documentation and configure it accordingly. For an example using Nginx, see https://help.mulesoft.com/s/article/Path-based-Nginx-Ingress-Routing-Example-for-Runtime-Fabric-on-Self-Managed-Kubernetes[Path-based Nginx Ingress Routing Example for Runtime Fabric on Self-Managed Kubernetes].
```
Comment (kevintroller) [inline modules/ROOT/pages/custom-ingress-configuration.adoc:117]: @digvijaysk19 @hegderohit89 can you add your clarification for the path based routing on line 113? 

Currently, states the following:

_<6> The `path` parameter value is replaced by the value you add in the *Path* field when you xref:configure-ingress-for-a-mule-application[configure ingress for a Mule application]. Note that `path` is not a placeholder._

**Comment by hegderohit89** [comment]
Screenshot 2024-12-16 at 5 18 14 PM
>

@kevintroller This is the image we need to refer

File: modules/ROOT/pages/custom-ingress-configuration.adoc [hunk 2]
```diff
@@ -110,64 +108,91 @@ If you add the TLS section with a `secretName` attribute, you must ensure that y
...
 
-<6> The `path` parameter value is replaced by the value you add in the *Path* field when you xref:configure-ingress-for-a-mule-application[configure ingress for a Mule application]. Note that `path` is not a placeholder. 
+<6> The `path` field is updated with the value specified in the *Path* field when configuring path-based routing for an xref:configure-ingress-for-a-mule-application[ingress] in your Mule application.(TODO:refer to the example).
+    Note that `path` is not a placeholder. 
 
 <7> These placeholder values are required for Kubernetes validation, but the actual values are not used by Runtime Fabric. 
 
-For a path-based routing ingress template, consult your ingress controller documentation and configure it accordingly. For an example using Nginx, see https://help.mulesoft.com/s/article/Path-based-Nginx-Ingress-Routing-Example-for-Runtime-Fabric-on-Self-Managed-Kubernetes[Path-based Nginx Ingress Routing Example for Runtime Fabric on Self-Managed Kubernetes].
-
-=== networking.k8s.io/v1beta1
```
Comment (hegderohit89) [inline modules/ROOT/pages/custom-ingress-configuration.adoc:119]: We no more support these old K8s versions, so its good one to remove

File: modules/ROOT/pages/custom-ingress-configuration.adoc (hunk 1 above)
Comment (hegderohit89) [reply modules/ROOT/pages/custom-ingress-configuration.adoc:117]: @digvijaysk19 can you please review and adjust if i have missed anything

**Review by dmerlob — DISMISSED** [review]

**Review by dmerlob — APPROVED** [review]

**Pull request merged**